src/uu/cksum/src/cksum.rs

```diff
@@ -253,7 +254,7 @@ where
                 }
             }
             _ => {
-                if options.untagged {
+                if options.untagged && !options.tag {
```

Comment:
This diff should not be able to matter, again because of `overrides_with`. I suggest reverting this, and the addition of the `tag: bool` field.

Unless of course I missed something, but then this code is wrong for a different reason.

Reply:
well spotted!